src/Cli/Microsoft.TemplateEngine.Cli/Commands/details/BaseDetailsCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Cli.NuGet;
+using Microsoft.TemplateEngine.Edge.Settings;
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal class BaseDetailsCommand : BaseCommand<DetailsCommandArgs>
```

Comment:
Base command is not needed in this case. Base commands were used only when we have 2 different commands doing exactly the same.
```suggestion
    internal class DetailsCommand : BaseCommand<DetailsCommandArgs>
```